cmd/mirror.go

```diff
@@ -81,12 +90,41 @@ func newMirrorSignCmd() *cobra.Command {
 				return cmd.Help()
 			}
 
-			if len(args) == 1 {
-				return v1manifest.SignManifestFile(args[0], env.Profile().Path(localdata.KeyInfoParentDir, "private.json"))
+			if privPath == "" {
+				privPath = env.Profile().Path(localdata.KeyInfoParentDir, "private.json")
```

Comment:
`private.json` has been used multiple times in the package, I think we should make it a constant value.